src/quic_solana.rs

```diff
@@ -115,16 +130,21 @@ impl ConnectionCache {
         }));
         let (identity_watcher, _) = watch::channel(initial_identity.pubkey());
         let (reactive_signer, _) = watch::channel(initial_signer);
+        let (stop_transactions, _) = watch::channel(());
```

Comment:
couldn't we use a send/receive channel here instead to indicate when reset can continue?